cms/envs/common.py

```diff
@@ -550,6 +550,9 @@
     'openedx.core.djangoapps.site_configuration.middleware.SessionCookieDomainOverrideMiddleware',
 ]
 
+# Edly Configuration
+ENABLE_EDLY_ORGANIZATIONS = 'enable_edly_organizations'
```

Comment:
`ENABLE_EDLY_ORGANIZATIONS_SWITCH`
Also mention this switch in PR description.
CC: @fayyazahmed66

Reply:
Added